### PR TITLE
Feat/clean deps

### PR DESCRIPTION
**Pull request by ShaswatPrabhat**
Please find the changes for the [issue](https://github.com/callstack/reassure/issues/54)

### CODE REVIEW

File: .changeset/silly-geckos-sip.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+'reassure': minor
```
Comment (mdjastrzebski) [inline .changeset/silly-geckos-sip.md:2]: It should be a patch update, as there is no visible change to the user here.

**Review by mdjastrzebski — CHANGES_REQUESTED** [review]
Looks good 👍🏻 @ShaswatPrabhat pls tweak the changeset from minor to patch level

File: .changeset/silly-geckos-sip.md (hunk 1 above)
Comment (ShaswatPrabhat) [reply .changeset/silly-geckos-sip.md]: Thanks, done @mdjastrzebski

**Review by mdjastrzebski — APPROVED** [review]
👍🏻

**Pull request merged**